src/routes/solid-router/reference/components/a.mdx

```diff
@@ -2,11 +2,35 @@
 title: A
 ---
 
-Like the [`<a>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) tag but supports relative paths and active class styling (requires client side JavaScript).
```

Comment:
If we are removing this link here should we add it in the next time <a> shows up?